site/content/blog/2019-12-20-onboarding-with-mattermost.md

```diff
@@ -0,0 +1,34 @@
+
+During the process, my mentor would help guide me in architecting the overall solution and in answering any questions I had. The community instance for Mattermost was also available, where I could post my questions to the core team and the open-source community. While I did not participate in any Scrum-related activities with the teams, I did follow their build and pull-request flow. The work overall was very fast-paced and responses to my questions and reviews of my PRs were surprisingly quick. Considering everyone is remote I was expecting the pace to be slower but was pleasantly surprised that I was helped and guided more quickly than I would have expected in a normal office environment.
+
+### Detailed Checklist and Follow-Up
+One of the first things that occurred was that my mentor and a member of Human Resources created an extensive checklist for me to follow. This took me through all the starting documentation I would need to know and introduced me to videos and our development processes. This took me through my first two-weeks and made it easy to keep on track as I learned more about my duties and the company as a whole. 
+
+### Extensive Resources
+There is a vast wealth of documentation available through Confluence. Everything from setting up your development environment and the Tick-Tock branching process to the setup and customization of the servers, apps and plugins. Many of these are part of the onboarding checklist, but it will take weeks to ingest all the knowledge!
+
+### Mentorship
+During my first few weeks I had daily mentors from different teams who were available for discussions and helped answer any questions that I had. This was a great way to build a bond with my peers and to learn more about the other teams. The Mattermost application extends across web, mobile, server side, Redux, and design; in meeting with the mentors from each team I was able to better understand how all the parts of the application come together.
+
+Beyond the daily mentorships, I also had my main mentor to always follow up with and help me through my daily tasks. One of the mindsets that was repeatedly conveyed was ‘Learn, Master, Teach’, where you learn a topic, master it, and then teach it to others, allowing the cycle to continue.  
+
+### Remote Environment
+One of the biggest draws of joining Mattermost is that everyone is remote, and the company has really built a culture around this. Every week there are multiple meetings that help create rapport among the team and instill the passion behind innovation and customer obsession. Unlike my previous remote environments, the environment that Mattermost has created makes you feel part of the team and not siloed off. There are even video meetings for hanging around the water cooler to just chat as well as a video game hour.
+
+### Overall
```

Comment:
```suggestion
### Summary
```

0/5 to be honest, but I thought 'summary' might fit better here as a section heading. Another heading I often see at the end of a blog is 'conclusion', but I thought you're not really 'concluding' anything here, per say.